src/guc.c

```diff
@@ -53,6 +81,11 @@ static const struct config_enum_entry loglevel_options[] = {
 	{ "log", LOG, false },		 { NULL, 0, false }
 };
 
+static const struct config_enum_entry transparent_decompression_options[] = {
+	{ "on", 1, false },	   { "true", 1, false },	   { "off", 0, false },
+	{ "false", 0, false }, { "hyperstore", 2, false }, { NULL, 0, false }
+};
+
```

Comment:
I forgot again what this thing does. So there are three values and two table types (compressed and TAM), maybe we should have a table on which value has which effect on which type of table.

Reply:
This change is to be able to run DecompressChunk even if the table/chunk is using the table access method. This is useful as a fallback in case there is a bug or there's an optimization that we don't implement that DecompressChunk has.

So, you can set the GUC to "hypestore" to enable this. The other options (on/off and true/false) are the existing options used to either enable or disable DecompressChunk on compressed tables.

Reply:
Let's add this as a comment?

Reply:
done.